app/views/parent_objects/_form.html.erb

```diff
@@ -63,7 +63,7 @@
 
   <div class="field">
     <%= form.label :visibility %>
-    <%= form.text_field :visibility %>
+    <%= form. select(:visibility,[['Private','Private'],['Public','Public'],['Yale Community Only','Yale Community Only']] ) %>
```

Comment:
I think you can declair these once `['Private', 'Public', 'Yale Community Only']` and Rails will automatically do the doubling. I could be wrong about that... if I am, then `ParentObject.visibilities.map {|v| [v,v] }` would work